xml/System/Console.xml

```diff
@@ -258,7 +258,7 @@
  :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.console.backgroundcolor/vb/backgroundcolor1.vb" id="Snippet1":::
 
  A get operation for a Windows-based application, in which a console does not exist, returns <xref:System.ConsoleColor.Black?displayProperty=nameWithType>.
-
+ On Linux, because there is no general mechanism to fetch the current Console colors, <xref:System.Console.BackgroundColor%2A> will return `(ConsoleColor)-1` until it has been set.
```

Comment:
It's not specific only to Linux, it applies to all Unixes. I would reword it in the following way:

```suggestion
 Unix systems don't provide any general mechanism to fetch the current console colors. Because of that, <xref:System.Console.BackgroundColor%2A> returns `(ConsoleColor)-1` until it is set in explicit way (using the setter).
```